querybook/webapp/components/DataTableViewOverview/DataTableViewOverview.tsx

```diff
@@ -133,6 +134,14 @@ export const DataTableViewOverview: React.FC<
     const tableLinksDOM = (table.table_links ?? []).map((link, index) => (
         <div key={index}>
             <Link to={link.url} newTab>
+                <Link2
+                    size={20}
+                    style={{
+                        position: 'relative',
+                        top: '5px',
+                        paddingRight: '5px',
+                    }}
```

Comment:
can you create a css class for it?

also, instead of using relative position, would it better to use a inline flexbox on the parent Link? like

{ display: 'inline-flex', alignItems: 'center', gap: '5px' }.

btw, it might be better to use times of 4px for all spaces

Reply:
+1, we are enforcing padding in multiples of 4px